libs/@guardian/atoms-rendering/src/YoutubeAtomOverlay.tsx

```diff
@@ -81,17 +84,46 @@ const playButtonStyling = css`
 	}
 `;
 
-const videoDurationStyles = css`
+const pillStyles = css`
 	position: absolute;
 	top: ${space[2]}px;
 	right: ${space[2]}px;
 	${textSans.xxsmall({ fontWeight: 'bold' })};
 	background-color: rgba(0, 0, 0, 0.7);
 	color: ${palette.neutral[100]};
-	padding: ${space[1]}px ${space[3]}px;
 	border-radius: ${space[3]}px;
+	padding: 0 6px;
+	display: inline-flex;
 `;
 
+const pillItemStyles = css`
+	:last-child:not(:only-of-type) {
+		border-left: 1px solid rgba(255, 255, 255, 0.5);
+	}
```

Comment:
Very clever! Might be worth adding a comment explaining what this does? Would `nth-of-type(n+2)` be more appropriate, as it targets all elements but the first.

What's `:not` support like these days?